tracee-ebpf/tracee/consts.go

```diff
@@ -79,6 +79,7 @@ const (
 	tailVfsWrite uint32 = iota
 	tailVfsWritev
 	tailSendBin
+	tailSendBinSyscall
```

Comment:
It's not exactly related to syscall, but raw_tracepoint bpf program type.
The other one is for kprobe program type.
So maybe tailSendBinTP will be a better name?